snuba/web/query.py

```diff
@@ -47,6 +48,24 @@ def __init__(
         self.meta = meta
 
 
+@dataclass
```

Comment:
```suggestion
@dataclass(frozen=True)
```

Should this be frozen? It looks like all of the parameters are set when it is instantiated and not mutated.